Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM node:18.17.0-alpine AS package
+FROM node:18.19.1-alpine AS package
```

Comment:
issue: you are missing a version bump down in this file. You just changed the build step.